boards/arm/nrf52840_pca10090/board.c

```diff
@@ -53,12 +55,18 @@ LOG_MODULE_REGISTER(board_control_pca10090);
  * | COEX2   | -- MCU Interface Pin 8 -- | P1.15    | COEX2_PH   |
  */
 
-/* The following tables specify the -default- values for each pin.
- * Thus, when configuring the pins if there is a one in this table,
- * pull the pin to zero instead.
+__packed struct pin_config {
+	u8_t pin;
+	u8_t val;
+};
+
+/* The following tables specify the default configuration of each pin.
+ * The switches have active-low logic, so when a pin is set to 1 in
+ * these tables, pull it low to leave it set to its default.
+ * When it is set to 0, pull it high to activate the non-default setting.
```

Comment:
I think this comment could be rephrased to be more comprehensible. Or is it just me that could not understand it despite reading it several times? Perhaps one of these "it" could be clarified, as well as the default stuff - it this "default configuration" and "non-default setting" referring to the same kind of thing?
Same applies to the very similar comment in the loop that configures the pins.

Reply:
Rewritten